portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  String[] rankings = {"VOSS", "FIJI", "SMART WATER", "CRYSTAL GEYSER"};
+  ArrayList<String> water_rankings = new ArrayList<String>();
+
+  public DataServlet() {
+      water_rankings.add("VOSS");
+      water_rankings.add("FIJI");
+      water_rankings.add("SMART WATER");
+      water_rankings.add("CRYSTAL GEYSER");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("Hello Maximilian!");
+    Gson gson = new Gson();
+    String output = gson.toJson(water_rankings); 
```

Comment:
I suggest write a new method named "getWaterRankings" which will return the list of strings, then in your doGet you convert those strings into gson objects.

Reply:
I did this, but converting an arraylist to json vs a normal array to json doesn't seem to cause any difference in what the javascript receives.